h2/src/main/org/h2/security/auth/ConfigProperties.java

```diff
@@ -28,7 +28,7 @@ public ConfigProperties(PropertyConfig... configProperties) {
 
     public ConfigProperties(Collection<PropertyConfig> configProperties) {
         properties = new HashMap<>();
-        if (properties != null) {
+        if (configProperties != null) {
```

Comment:
wrong

Reply:
Can you please elaborate? As I can see `properties ` definitely can't be `null` here. 
So it doesn't make sense to check it.
Also I found that in this constructor - it's possible to pass `null` as `configProperties`
```
    public ConfigProperties(PropertyConfig... configProperties) {
        this(configProperties == null ? null : Arrays.asList(configProperties));
    }
```
I believe my proposed fix is correct

Reply:
Sorry, you are correct